guides/common/modules/snip_red-hat-helper-apps.adoc

```diff
@@ -0,0 +1,11 @@
+ifdef::upgrading-connected,upgrading-disconnected[]
+For interactive upgrade instructions, you can use the {ProjectName} Upgrade Helper on the Red{nbsp}Hat Customer Portal.
+This application provides you with an exact guide to match your current version number.
+You can find instructions that are specific to your upgrade path, as well as steps to prevent known issues.
+For more information, see https://access.redhat.com/labs/satelliteupgradehelper/[{Project} Upgrade Helper] on the Red{nbsp}Hat Customer Portal.
+endif::[]
+ifdef::installing-satellite-server-connected,installing-satellite-server-disconnected,installing-capsule-server[]
+For interactive installation instructions, you can use the {ProjectName} Installation Helper on the Red{nbsp}Hat Customer Portal.
+This application provides you with an interactive way to prepare installation instructions to match your required {Project} version number and configuration.
+For more information, see https://access.redhat.com/labs/rhsih/[{ProjectName} Installation Helper] on the Red{nbsp}Hat Customer Portal.
+endif::[]
```

Comment:
This is based on the old lines in con_upgrading-overview.adoc, just copied over to a snippet and duplicated for the new installation app.